### PR TITLE
Chatroom css

### PR DESCRIPTION
**Pull request by TinyGobby**

### CODE REVIEW

**Comment by TinyGobby** [comment]
[Chatroom CSS](https://trello.com/c/2tmGQkPe/117-chatroom-css)

**Comment by TinyGobby** [comment]
[Change Home to Form](https://trello.com/c/fe6ctXGj/119-change-home-to-form)

File: frontend/src/chatroom.js [hunk 1]
```diff
@@ -86,34 +86,38 @@ class ChatRoom extends Component {
...
+            <h3>Fake Names</h3>
+            <ShowFakeNames fakeNames={this.state.fakeNames} />
+          </div>
+          <div className={styles.singleNameDiv}>
+            <h3>Real Names</h3>
+            <ShowRealNames realNames={this.state.realNames} />
+          </div>
         </div>
-        <div>
-          <Guess guesser={this.props.user} />
```
Comment (sebgrebe) [inline frontend/src/chatroom.js]: moved guesser to top to make sure user can see success/error message.

File: frontend/src/chatroom.js [hunk 2]
```diff
@@ -86,34 +86,38 @@ class ChatRoom extends Component {
...
+              name="message"
+              className={styles.messageInput}
+              value={this.state.input}
+              onChange={this.handleChange}
+              placeholder="Type something..."
+            />
+            <button className={styles.button} type="submit">
+              Submit!
+            </button>
+          </form>
```
Comment (sebgrebe) [inline frontend/src/chatroom.js]: would be good to make this a component

File: frontend/src/fakeNames.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ class ShowFakeNames extends Component {
           return (
             <div>
               <ul>
-                <li className="fakeName">{element}</li>
+                <li className={styles.names}>{element}</li>
```
Comment (sebgrebe) [inline frontend/src/fakeNames.js]: Nice job on having different css files and importing them. Improves the structure

File: frontend/src/home.js [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
   constructor(props) {
     super(props);
   }
   render() {
     return (
-      <div>
-        <h1 className="App-title">Guess Vu!</h1>
-        <Form setUser={this.props.setUser}/>
+      <div className={styles.homepage}>
+        <img className={styles.image} src={require('../styles/my-logo-stark.png')} alt="logo" />
```
Comment (sebgrebe) [inline frontend/src/home.js]: Love the logo!

File: frontend/styles/app.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.container {
+  padding: 10px 20px 10px 20px;
```
Comment (sebgrebe) [inline frontend/styles/app.css]: added padding on the main container.

File: test/features/app.test.js [hunk 6]
```diff
@@ -23,7 +23,7 @@ describe('Guess Vu', () => {
 
   describe('Name form', () => {
     test('redirects to chatroom after signup', async () => {
-      await page.waitForSelector('.Form');
+      await page.waitForSelector('#Form');
```
Comment (sebgrebe) [inline test/features/app.test.js]: Sure this works? Let's leave it for now though, all of the tests need to be fixed to adjust for imported styles. Let's make a ticket for this.

**Pull request merged**